src/components/editpage/BreadCrumb.js

```diff
@@ -0,0 +1,59 @@
+import { validation } from "../../validation.js";
+
+export default function BreadCrumb({ $target, initialState, clickPath }) {
+  validation(new.target, "BreadCrumb");
+
+  const $breadCrumb = document.createElement("nav");
+  $breadCrumb.className = "linkContainer";
+  $target.appendChild($breadCrumb);
+
+  this.state = initialState;
+
+  this.setState = (nextState) => {
+    this.state = nextState;
+    this.render();
+  };
+
+  this.renderBreadCrumb = (id) => {
+    const path = [];
+
+    const recursiveBC = (id) => {
+      const $curLi = document.getElementById(id);
+      const $ul = $curLi.closest("ul");
+
+      if ($ul.className === "child") {
+        path.push([$curLi.querySelector("span").innerHTML, id]);
+
```

Comment:
```suggestion
      const $curLi = document.getElementById(id);
      if ($curLi) {
        const $ul = $curLi.closest("ul");
        if ($ul.className === "child") {
          path.push([$curLi.querySelector("span").innerHTML, id]);

```
$curLi가 존재하는가에 대한 방어코드를 넣는것도 좋을듯 합니다

Reply:
해당 컴포넌트에서는 방어코드가 없었는데 중요한 부분을 잘 잡아주신것 같습니다. 리뷰 반영해보도록 하겠습니다! 감사합니다👍